data/data/openstack/topology/private-network.tf

```diff
@@ -3,18 +3,25 @@ locals {
   service_cidr_block = "${cidrsubnet(var.cidr_block, 1, 1)}"
 }
 
+data "openstack_networking_network_v2" "external_network" {
+  name       = "${var.external_network}"
```

Comment:
Do here need prefix?

Reply:
no, this is a query to ge the existing, shared, external network. It's not creating a resource.

Reply:
oh, got you